src/app/Fake.DotNet.NuGet/NuGet.fs

```diff
@@ -39,6 +39,18 @@ type NugetSymbolPackage =
     /// Build a symbol package using the nuspec file
     | Nuspec = 2
 
+type ToolOptions =
```

Comment:
Can you add short docs for this or mark it `internal`/`private`?

Reply:
I'll make it internal, since I'm not reusing it in the DotNet module anymore.